src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -951,15 +943,170 @@ The description of the configuration options for the remaining ProActive compone
+   -o              (optional) Enable offline mode. Can be used with -i, -a, or -p
+   -u              (optional) Enable upgrade mode. Can be used with -i, -a, or -p and makes -admin_pwd and -user_pwd optional
+
+ Examples:
+   sudo ./proactiveInstaller.sh -i
+   sudo ./proactiveInstaller.sh -i -o
+   sudo ./proactiveInstaller.sh -i -u
+   sudo ./proactiveInstaller.sh -i -o -u
+   sudo ./proactiveInstaller.sh -a -f ./password-template.properties
+   sudo ./proactiveInstaller.sh -a -f ./password-template.properties -o
+   sudo ./proactiveInstaller.sh -a -f ./password-template.properties -u
+   sudo ./proactiveInstaller.sh -a -f ./password-template.properties -o -u
+   sudo ./proactiveInstaller.sh -p 1234 -admin_pwd adminPass123 -user_pwd userPass456
+   sudo ./proactiveInstaller.sh -p 1234 -admin_pwd adminPass123 -user_pwd userPass456 -o
+   sudo ./proactiveInstaller.sh -p 1234 -u
+   sudo ./proactiveInstaller.sh -p 1234 -u -o
+----
+
+In the following sections, we will present the three methods you can use to install the ProActive server:
+
+-  **CLI Arguments Mode**: Passwords provided directly through command-line arguments for a minimal and non-interactive installation.
+-  **Interactive Mode**: Step-by-step guidance for users to input passwords.
+-  **Automatic Mode**: Installation using a configuration file with predefined passwords.
+
+===== ProActive server installation Using CLI Password Arguments
+
+This method provides a simple and minimal installation of the ProActive server without supporting advanced configurations (such as LDAP, Users, TLS, or database setup).
+
+It installs the required configuration of the ProActive server by passing passwords as command-line arguments. This allows you to supply the required credentials directly via the command line, avoiding the need for interactive password input during installation.
+
+During this mode, you must provide a valid passphrase for password encryption using the `-p` option, followed by the two required passwords for a basic installation: the admin login password (`-admin_pwd`) and the user password (`-user_pwd`).
+
+The ProActive installer will use the provided passphrase to encrypt the passwords and insert them, in encrypted format, into the `inputConfig.json` file.
+
+Example:
+----
+$ sudo ./proactiveInstaller.sh -p MySecurePassphrase -admin_pwd AdminPass123 -user_pwd UserPass456
+
+ CLI installation mode is set.
+ Running in CLI installation mode.
+...
+----
+
+If you want to start the ProActive server installation and configuration offline (without requiring internet access), you can include the `-o` option in the installation command to ensure it runs in offline mode:
+----
+$ sudo ./proactiveInstaller.sh -p MySecurePassphrase -admin_pwd AdminPass123 -user_pwd UserPass456 -o
+----
+
+NOTE: It is important to ensure that the command-line interface where you provide the passwords is secure, as passwords may be visible in the process list.
```

Comment:
Maybe like this:

Note: Avoid passing sensitive information as CLI arguments in shared environments, as they may be visible in the process list.